clippy_lints/src/functions/mod.rs

```diff
@@ -326,6 +327,25 @@ declare_clippy_lint! {
     "getter method returning the wrong field"
 }
 
+declare_clippy_lint! {
+    /// ### What it does
+    /// Detects lifetimes that are secretly `'static`.
```

Comment:
My two cents: The language "secretly" is extremely vague. Maybe: "Detects generic lifetimes that the compiler will always resolve to be `'static`"

Reply:
Suggestion applied.